source/ZividNETSamples.sln

```diff
@@ -42,92 +44,99 @@ EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Warmup", "Camera\InfoUtilOther\Warmup\Warmup.csproj", "{B2F9FB41-99ED-4DD2-8683-02D325997619}"
 EndProject
 Global
-    GlobalSection(SolutionConfigurationPlatforms) = preSolution
```

Comment:
I don't understand why this change happens? @knatten @apartridge do you know?

Reply:
https://stackoverflow.com/a/44209332